lib/core/versions/latest/RequestHandler.ts

```diff
@@ -74,7 +75,17 @@ export default class RequestHandler implements IRequestHandler {
       switch (operation.type) {
         case OperationType.Create:
 
-          const document = await this.applyCreateOperation(operation);
+          const documentState = await this.applyCreateOperation(operation);
```

Comment:
Maybe we can do the transformation in the `handleResolveRequest` function. Seems like that logic is common to both type of resolutions: longForm and did.

Reply:
Great suggestion. Done.